base/cvd/cuttlefish/host/commands/cvd/server_command/status_fetcher.cpp

```diff
@@ -224,10 +224,28 @@ Result<StatusFetcherOutput> StatusFetcher::FetchStatus(
 }
 
 Result<Json::Value> StatusFetcher::FetchGroupStatus(
-    const selector::LocalInstanceGroup& group,
+    selector::LocalInstanceGroup& group,
     const RequestWithStdio& original_request) {
   Json::Value group_json(Json::objectValue);
   group_json["group_name"] = group.GroupName();
+  std::set<cvd::InstanceState> reachable_states{
+      cvd::InstanceState::RUNNING, cvd::InstanceState::STARTING,
+      cvd::InstanceState::UNREACHABLE};
```

Comment:
It's odd naming to have `UNREACHABLE` as a member of `reachable_states`.

Maybe `states_to_message`?